js/upgrade.js

```diff
@@ -320,14 +320,18 @@ function startProcess(type) {
 }
 
 function afterUpgradeNow(res) {
-  startProcess("upgrade");
-  $("#upgradeNow")
-    .unbind()
-    .replaceWith(
-      "<span id=\"upgradeNow\" class=\"button-autoupgrade\">"
-      + input.translation.upgradingPrestaShop
-      + " ...</span>"
-    );
+
+  $('#confirm-upgrade').modal()
+    .on('click', '#confirm-upgrade', function(e) {
+      startProcess("upgrade");
```

Comment:
```suggestion
      startProcess('upgrade');
```

Do not mix single and double quotes

Reply:
yes, fixing it (that's legacy code that was only moved)